test/component/testDeclarationPageMultipleApplicants.js

```diff
@@ -172,161 +188,341 @@ describe('declaration, multiple applicants', () => {
+            contentData.applicantCurrentName = 'Bob Smith';
+            contentData.applicantWillName = 'larry bird';
+            contentData.aliasReason = 'got divorced';
+            contentData.detailsOfApplicants = 'Bob Smith of flat 1, somewhere rd, nowhere., fname1 sname1 of 1 qwe\r\n1 asd\r\n1 zxc and fname4 sname4 of 4 qwe\r\n4 asd\r\n4 zxc';
+            contentData.applicantCurrentNameSign = [
+                'fname1 sname1',
+                'fname4 sname4'
+            ];
 
             testWrapper.agent.post('/prepare-session/form')
                 .send(sessionData)
                 .end(() => {
-                    testWrapper.agent.get(testWrapper.pageUrl)
-                        .then(response => {
-                            assert(response.text.includes('an applicant, an executor named in the will as Bob Alias, is applying for probate. Their name is different because an applicant got divorced.'));
-                            assert(response.text.includes('fname1 sname1, an executor named in the will, is applying for probate.'));
-                            assert(response.text.includes('fname4 sname4, an executor named in the will, is applying for probate.'));
-                            done();
-                        })
-                        .catch(err => {
-                            done(err);
-                        });
+                    testWrapper.testContent(done, contentToExclude, contentData);
                 });
         });
 
         it('test right content loaded on the page, applicant has an alias, one other executor has a different name (reason given as: Divorce), deceased has no other names and there are codicils', (done) => {
-            sessionData.will.codicils = 'Yes';
-            sessionData.executors.list[0].nameAsOnTheWill = 'No';
-            sessionData.executors.list[0].alias = 'Bob Alias';
+            const contentToExclude = [
+                'deceasedOtherNames',
+                'executorApplyingName',
+                'executorNotApplyingHeader',
+                'executorNotApplyingReason',
+                'executorNotApplyingReason-codicils',
+                'optionDiedBefore',
+                'optionDiedAfter',
+                'optionPowerReserved',
+                'optionRenunciated',
+                'additionalExecutorNotified',
+                'intro',
+                'legalStatementApplicant',
+                'deceasedEstateLand',
+                'applicantName',
+                'applicantName-codicils',
+                'applicantName-multipleApplicants',
+                'applicantName-multipleApplicants-mainApplicant-codicils',
+                'applicantName-multipleApplicants-mainApplicant-alias-codicils',
+                'applicantName-multipleApplicants-mainApplicant',
+                'applicantName-multipleApplicants-codicils',
+                'applicantName-multipleApplicants-alias-codicils',
```

Comment:
In this test an other applicant has an alias and there are codicils so perhaps we should be including this content?

`'applicantName-multipleApplicants-alias-codicils'`